src/config/layers.json

```diff
@@ -162,5 +162,108 @@
       {"value": 30, "color": "#1d2e83"},
       {"value": "255 - no value", "color": "#ffffff", "alpha": 0}
     ]
-  }        
-}  
\ No newline at end of file
+  },
+  "mxd13c1_vim_dekad": {
```

Comment:
can we make these names a bit more readable than the server_layer_name if it's easy?